builtin/providers/aws/resource_aws_dms_replication_task.go

```diff
@@ -0,0 +1,369 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/private/waiter"
+	dms "github.com/aws/aws-sdk-go/service/databasemigrationservice"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsDmsReplicationTask() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDmsReplicationTaskCreate,
+		Read:   resourceAwsDmsReplicationTaskRead,
+		Update: resourceAwsDmsReplicationTaskUpdate,
+		Delete: resourceAwsDmsReplicationTaskDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"cdc_start_time": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				// Unix timestamp 1484346880
```

Comment:
Do you just want me to add context? The API documentation was unclear what type of timestamp this should be.